scripts/ff_ci_pr_build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
It would be nice if this had more documentation describing what it does in comments at the top. `ff` is not a common acronym.